policies/api/openapi.yaml

```diff
@@ -21,7 +21,7 @@ paths:
           content:
```

Comment:
Use the openapi PR to this change... rollback the openapi.yaml file as the develop with the command below
❯ git checkout origin/develop policies/api/openapi.yaml